osc/build.py

```diff
@@ -1320,6 +1322,12 @@ def __str__(self):
                         print("Using prefered package: " + path + "/" + filename)
                         os.unlink(tffn)
 
+        if bi.containerannotation:
+            if not os.path.exists("containers"):
+                os.makedirs("containers")
+            with open("containers/annotation", "wb") as f:
+                f.write(bi.containerannotation.encode())
```

Comment:
Wouldn't be better to avoid encoding to bytes?
```
            with open("containers/annotation", "w", encoding="utf-8") as f:
                f.write(bi.containerannotation)
```